src/main/java/org/pentaho/metastore/stores/xml/BaseXmlMetaStore.java

```diff
@@ -61,12 +74,23 @@ protected static XmlMetaStoreCache defaultCache() {
   }
 
   /**
-   * @return the rootFolder
+   * @return the rootFolder in use
    */
   public String getRootFolder() {
     return rootFolder;
   }
 
+  /**
+   * Set the root folder. This will be the exact path that the metastore will operate in (namespaces will be created
+   * immediately below this). It will not have XmlUtil.META_FOLDER_NAME appended.
+   *
+   *
+   * @param rootFolder
+   */
+  public void setRootFolder( String rootFolder ) {
```

Comment:
We will need to verify that the PentahoServer can export the metastore with our change

Reply:
We haven't done any other testing with pentaho server, yet. Maybe we should do that testing in the Pentaho Server POC story? 
although, is export even a requirement for that story?